server/methods/accounts/accounts.js

```diff
@@ -1,3 +1,4 @@
+import { Meteor } from "meteor/meteor";
```

Comment:
I've made changes to the import order on this file, but it doesn't seem to be reflecting on this comment.
https://github.com/reactioncommerce/reaction/blob/michael-verify-email-address-2125/server/methods/accounts/accounts.js